docs/mkdocs.yml

```
@@ -1,4 +1,4 @@
-site_name: Terraform for Atlassian DC on Kubernetes
+site_name: Deployment Automation for Atlassian DC on K8s
```

```suggestion
site_name: Deployment Automation for Atlassian DC on Kubernetes
```

I prever the longer version, at least here for SEO.